origin-bridge/src/controllers/twitter.js

```diff
@@ -0,0 +1,119 @@
+'use strict'
+
+const express = require('express')
+const router = express.Router()
+
+const {
+  getTwitterOAuthRequestToken,
+  getTwitterOAuthAccessToken,
+  verifyTwitterCredentials
+} = require('../utils/twitter')
+const { generateAttestation } = require('../utils/attestation')
+const { mapObjectToQueryParams } = require('../utils')
+const { twitterVerifyCode } = require('../utils/validation')
+const constants = require('../constants')
+const Attestation = require('../models/index').Attestation
+const AttestationTypes = Attestation.AttestationTypes
+const logger = require('../logger')
+
+/* Get an oAuth request token from Twitter.
+ *
+ */
+router.get('/auth-url', async (req, res) => {
+  const dappRedirectUrl = req.query.dappRedirectUrl || null
+
+  let oAuthToken, oAuthTokenSecret
+  try {
+    // eslint-disable-next-line no-extra-semi
+    ;({ oAuthToken, oAuthTokenSecret } = await getTwitterOAuthRequestToken(
```

Comment:
nit: why is the leading ; and the () needed ?
Couldn't this be written without them as:
` { oAuthToken, oAuthTokenSecret } = await getTwitterOAuthRequestToken(dappRedirectUrl)`

Reply:
That syntax isn't valid for destructuring objects, it only works if you also are declaring, e.g. `let { oAuthToken, oAuthTokenSecret } = await getTwitterOAuthRequestToken(dappRedirectUrl)` would work. It would work without the leading `;` but that gets added by prettier. Linting then fails because of an unnecessary semicolon which is why the eslint-disable is needed.

Crazy right? :grin:

Reply:
Haha what a face plant by the linter/prettifier